json-rpc/Cargo.toml

```diff
@@ -57,6 +60,8 @@ executor = { path = "../execution/executor", version = "0.1.0" }
 executor-types = { path = "../execution/executor-types", version = "0.1.0" }
 scratchpad = { path = "../storage/scratchpad", version = "0.1.0" }
 move-vm-types = { path = "../language/move-vm/types", version = "0.1.0" }
+transaction-builder-generated = { path = "../client/transaction-builder", version = "0.1.0" }
+workspace-builder = { path = "../common/workspace-builder", version = "0.1.0" }
```

Comment:
Please avoid using this dependency. I have a task in my backlog to remove this as it is harmful to the developer experience.